kubernetes/loculus/values.schema.json

```diff
@@ -0,0 +1,998 @@
+{
+  "title": "values.yaml Schema for the Loculus Helm Chart",
+  "description": "Also has 'groups' and 'placeholder' keys in some places, which are used during docs generation. You can set 'docsIncludePrefix' to false to cut of a path when generating docs. Have a look at the SchemaDocs.astro component in the Loculus docs.",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "metadata": {
```

Comment:
<img width="775" alt="image" src="https://github.com/user-attachments/assets/e38b7a15-80c4-44da-b66f-d8e776b541df" />
would be clearer if it was about a `Metadata item` or similar (metadata is plural)

Reply:
Maybe "Metadata field"?

Reply:
https://github.com/loculus-project/loculus/pull/3588/commits/8905db7f6628fcf9650c371719cd8bdbfc1f46c3